.github/workflows/main.yml

```diff
@@ -64,20 +65,18 @@ jobs:
         id: meta
         uses: docker/metadata-action@v3
         with:
-          images: |
-            chirpstack/chirpstack-application-server
+          images: chirpstack/${{ github.event.repository.name }}
           tags: |
+            type=ref,event=branch
```

Comment:
I think I would like to not push branches, only tags. In case there is a test version, I tag a vX.Y.Z-test.X release. I think the `type=ref,event=branch` line is therefore not needed.

Reply:
updated